zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -20,21 +20,31 @@
 import static java.lang.String.format;
 
 import java.io.IOException;
-import java.util.*;
+import java.util.Date;
```

Comment:
Zeppelin need to have a unified code style (whether import * or import each separately), would be appreciated someone can do that (adding checkstyle)